### PR TITLE
Update client binary version from 4.1.x to 4.3.x, remove rhcos-image

### PR DESCRIPTION
**Pull request by praveenkumar**

### CODE REVIEW

File: provision.sh [hunk 1]
```diff
@@ -107,13 +107,13 @@ update-rhcos-image
 
 echo "Installing oc client"
 cd $HOME
-curl -OL https://mirror.openshift.com/pub/openshift-v4/clients/ocp/4.1.4/openshift-client-linux-4.1.4.tar.gz
-tar -zxf openshift-client-linux-4.1.4.tar.gz
-rm -fr openshift-client-linux-4.1.4.tar.gz
+curl -OL https://mirror.openshift.com/pub/openshift-v4/clients/ocp/4.3.0/openshift-client-linux-4.3.0.tar.gz
```
Comment (cfergeau) [inline provision.sh:107]: Probably best to use https://mirror.openshift.com/pub/openshift-v4/clients/ocp/stable-4.3/

File: provision.sh [hunk 2]
```diff
@@ -107,13 +107,13 @@ update-rhcos-image
...
+tar -zxf openshift-client-linux-4.3.0.tar.gz
+rm -fr openshift-client-linux-4.3.0.tar.gz
 sudo mv $HOME/oc /usr/local/bin
 
 echo "Installing kubectl binary"
-sudo ln -s /usr/local/bin/oc /usr/local/bin/kubectl
+sudo mv $HOME/kubectl /usr/local/bin
```
Comment (cfergeau) [inline provision.sh:113]: Is this related to this PR?

File: provision.sh [hunk 3]
```diff
@@ -102,9 +102,6 @@ EOF
 sudo virsh pool-start default
 sudo virsh pool-autostart default
 
-# Install a default RHCOS image
-update-rhcos-image
-
```
Comment (cfergeau) [inline provision.sh:107]: Can  openshift4-libvirt-gcp/tools/update-rhcos-image  be removed?

File: tools/create-cluster [hunk 4]
```diff
@@ -58,6 +58,10 @@ EOF
 openshift-install --dir="$CLUSTER_DIR" create manifests
 # Workaround for https://github.com/openshift/installer/issues/1007
 yq w -i $CLUSTER_DIR/manifests/cluster-ingress-02-config.yml spec[domain] apps.$BASE_DOMAIN
+yq w -i $CLUSTER_DIR/openshift/99_openshift-cluster-api_master-machines-0.yaml spec.providerSpec.value[domainMemory] 10192
+yq w -i $CLUSTER_DIR/openshift/99_openshift-cluster-api_master-machines-0.yaml spec.providerSpec.value[domainVcpu] 4
+yq w -i $CLUSTER_DIR/openshift/99_openshift-cluster-api_worker-machineset-0.yaml spec.template.spec.providerSpec.value[domainMemory] 10192
+yq w -i $CLUSTER_DIR/openshift/99_openshift-cluster-api_worker-machineset-0.yaml spec.template.spec.providerSpec.value[domainVcpu] 6
+
 
-export TF_VAR_libvirt_master_memory=11024
```
Comment (cfergeau) [inline tools/create-cluster:62]: I'd add a "why" this needs to be changed in the commit log. I'm also not sure it's worth changing from 11024 to 10192? And the vcpu stuff is something new, maybe better to split it or remove it.

**Comment by sallyom** [comment]
This was implemented in rhel8 branch, and we're settting rhel8 to master.
